Documentation/ApiOverview/Backend/JavaScript/ES6/_JavaScriptModulesExtended.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+    'dependencies' => ['core', 'backend'],
+    'imports' => [
+        '@vendor/my-extension/' => [
+            'path' => 'EXT:my_extension/Resources/Public/JavaScript/',
+            // Exclude files of the following folders from being import-mapped
+            'exclude' => [
+                'EXT:my_extension/Resources/Public/JavaScript/Contrib/',
+                'EXT:my_extension/Resources/Public/JavaScript/Overrides/',
+            ],
+        ],
+        // Adding a third party package
+        'thirdpartypkg' => 'EXT:my_extension/Resources/Public/JavaScript/Contrib/thidpartypkg/index.js',
+        'thidpartypkg/' => 'EXT:my_extension/Resources/Public/JavaScript/Contrib/thirdpartypkg/',
```

Comment:
```suggestion
        'thirdpartypkg/' => 'EXT:my_extension/Resources/Public/JavaScript/Contrib/thirdpartypkg/',
```

Reply:
Then the key would be the same like above. Also the example comes like that from the changelog, you are sure about this change?